crates/biome_rowan/src/token_text.rs

```diff
@@ -57,6 +57,11 @@ impl TokenText {
         self.range.is_empty()
     }
 
+    /// Returns the range of the text
```

Comment:
You might want to be more specific, because the range of a text can contain trivia too. Is that the case? We should specify it